apps/android/package.json

```diff
@@ -19,7 +19,7 @@
     "android": "react-native run-android "
   },
   "dependencies": {
-    "@fluentui-react-native/tester": "^0.24.2",
+    "@fluentui-react-native/tester": "^0.25.1",
```

Comment:
I would encourage you to do the following to lower the amount of diffs that we get in this PR:

`git checkout activityindicator_prototype`
`git pull upstream master`
`git push`

Reply:
Actually I just did it. The purpose was that `activityindicator_prototype` was out of sync with master but your branch wasn't, so your PR looked like it had a lot more changes than it did. After syncing it, the diff went from 184 files to 14 :')

Reply:
Great, thank you!